CocktailRecipes/CocktailRecipes/Features/Home/Charts/Views/DrinkChartCell.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  DrinkChartCell.swift
+//  CocktailRecipes
+//
+//  Created by Jaison Vieira on 10/17/20.
+//  Copyright © 2020 Eric Rado. All rights reserved.
+//
+
+import UIKit
+
+final class DrinkChartCell: UICollectionViewCell {
```

Comment:
I think we can reuse `SmallDrinkCell` and just add `rankLabel` to it

Reply:
It's done already. This is an outdated file. It was removed on my last commit.